src/test/groovy/com/auth0/gradle/oss/AndroidLibraryPluginTest.groovy

```diff
@@ -5,12 +5,19 @@ import org.gradle.testfixtures.ProjectBuilder
 import spock.lang.Specification
 import spock.lang.Unroll
 
+import java.nio.file.Files
+
 class AndroidLibraryPluginTest extends Specification {
 
     Project project
 
     void setup() {
         project = ProjectBuilder.builder().build()
+        Files.copy(
+                new File("local.properties").toPath(),
```

Comment:
Why is this required?

Reply:
Because the Android Gradle Plugin needs to know where the Android SDK is. These tests are broken on master.